src/Controllers/CalculatorController.cpp

```diff
@@ -29,81 +31,65 @@ static void sendError(Pistache::Http::ResponseWriter& res, std::string message)
 
 void CalculatorController::calculate(const Pistache::Rest::Request& req, Pistache::Http::ResponseWriter res)
 {
-    std::string bodyString = req.body();
-    rapidjson::Document bodyJson;
-    bodyJson.Parse(bodyString.c_str());
-    if (!bodyJson.IsObject())
+    std::string equationSchema;
+    bool schemaLoaded = flatbuffers::LoadFile("./flatbuffers/Equation.fbs", false, &equationSchema);
+    if (!schemaLoaded)
     {
-        sendError(res, "Root value is not a valid object.");
+        res.send(
+            Pistache::Http::Code::Internal_Server_Error,
+            "Unable to initialize parsing",
+            Pistache::Http::Mime::MediaType(
+                Pistache::Http::Mime::Type::Text,
+                Pistache::Http::Mime::Subtype::Plain));
         return;
     }
 
-    if (!bodyJson.HasMember("equation"))
+    flatbuffers::Parser parser;
+    bool schemaParsed = parser.Parse(equationSchema.c_str());
+    if (!schemaParsed)
     {
-        sendError(res, "Request body should be a JSON object with an \"equation\" string.");
+        res.send(
+            Pistache::Http::Code::Internal_Server_Error,
+            "Unable to initialize equation parsing",
+            Pistache::Http::Mime::MediaType(
+                Pistache::Http::Mime::Type::Text,
+                Pistache::Http::Mime::Subtype::Plain));
         return;
     }
-    const rapidjson::Value& equationJson = bodyJson["equation"];
-    if (!equationJson.IsString())
+    bool jsonParsed = parser.Parse(req.body().c_str());
```

Comment:
Nice use of the flatbuffer parsing. Having flatbuffers convert to/from .json for you can save a lot of code.